changelogs/unreleased/info-cluster-rename.md

```diff
@@ -0,0 +1,4 @@
+## bugfix/core
+
+* **[Breaking change]** The table `box.info.cluster` is renamed to
```

Comment:
AFAIU we should use the `compat` module to avoid breaking changes like this.

I think we could introduce a new `compat` option, set it to "new" by default (because the change is done in the major version), and show the `box.info.cluster` entry (along with `box.info.replication`) if it's set to "old".

Up to @Totktonada.

Reply:
If it is not hard to implement (I guess so), it is good to have a compat option to allow a user to postpone changes in, say, some monitoring code.

Reply:
I added a compat option. I didn't add it in the first version, because the RFC explicitly saying *"without a compat option"* was approved.